### PR TITLE
feat: add Spring Boot service

### PR DESCRIPTION
**Pull request by genryxy**
Part of #47
Create a simple Spring Boot service for uploading and downloading files. This service will be used lased to compare performance with Artipie realization.

### CODE REVIEW

File: spring-file/src/main/java/com/example/file/FileController.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+public class FileController {
```
Comment (olenagerasimova) [inline spring-file/src/main/java/com/example/file/FileController.java]: what will happen if runtime exception is thrown while saving/getting file? 500 status will be returned?

File: spring-file/src/main/java/com/example/file/FileController.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private FileService fileserv;
+
+    @Autowired
+    public FileController(final FileService fileserv) {
+        this.fileserv = fileserv;
+    }
+
+    @PostMapping("upload")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ResponseEntity<String> uploadFile(@RequestParam("file") MultipartFile file) {
```
Comment (olenagerasimova) [inline spring-file/src/main/java/com/example/file/FileController.java]: why multipart? in our files adapter files are uploaded not as multipart body, file binary body is the body of the request...

File: spring-file/src/main/java/com/example/file/FileController.java (hunk 1 above)
Comment (olenagerasimova) [inline spring-file/src/main/java/com/example/file/FileController.java]: also, I'd suggest to add 
- unit tests for the `FileService` and controller
- qulice to check codestyle

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, please, check my comments

File: spring-file/src/main/java/com/example/file/FileController.java (hunk 1 above)
Comment (genryxy) [reply spring-file/src/main/java/com/example/file/FileController.java]: Is it necessary to add qulice check style if the main purpose of this application is only benchmark?

Comment (olenagerasimova) [reply spring-file/src/main/java/com/example/file/FileController.java]: I'd say yes, it's even more important for benchmarks to be properly written and perfectly readable as in the case of mistake the results can be not valid... It's my opinion, however, ask Kirill if you have doubts.

**Comment by genryxy** [comment]
@olenagerasimova thanks, all corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good to me

**Comment by genryxy** [comment]
@g4s8 please check/merge

**Pull request merged**